scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -960,8 +960,8 @@ sealed trait SCollection[T] extends PCollectionWrapper[T] {
    * @group output
    */
   def saveAsTextFile(path: String,
-                     suffix: String = ".txt",
                      numShards: Int = 0,
+                     suffix: String = ".txt",
```

Comment:
This is technically breaking but it'll be a breaking change anyway.
It's to make the API consistent with other `saveAs*` methods.

Reply:
@nevillelyh I'm ok with that!